Depend-on/Bash.sh

```diff
@@ -0,0 +1,221 @@
+#!/bin/bash
+
+identify_and_run() {
+    os=$(uname)
+    if [ "$os" = "Linux" ]; then
+        distro=$(lsb_release -si)
+        case "$distro" in
+            CentOS|RedHatEnterpriseServer)
+                if command -v dnf &>/dev/null; then
+                    echo "The following packages will be installed:"
+                    echo "FFmpeg"
+                    echo "libavcodec-devel libavformat-devel libavutil-devel libswscale-devel"
+                    echo "freerdp2-devel"
+                    echo "pango-devel"
+                    echo "libssh2-devel"
+                    echo "libtelnet-devel"
+                    echo "libvncserver-devel"
+                    echo "libwebsockets-devel"
+                    echo "pulseaudio-libs-devel"
+                    echo "openssl-devel"
+                    echo "libvorbis-devel"
+                    echo "libwebp-devel"
+
+                    read -p "Do you understand? (yes/no): " choice
+                    case "$choice" in 
+                        yes|YES|y|Y)
+                            sudo dnf install -y FFmpeg
+                            sudo dnf install -y libavcodec-devel libavformat-devel libavutil-devel libswscale-devel
+                            sudo dnf install -y freerdp2-devel
+                            sudo dnf install -y pango-devel
+                            sudo dnf install -y libssh2-devel
+                            sudo dnf install -y libtelnet-devel
+                            sudo dnf install -y libvncserver-devel
+                            sudo dnf install -y libwebsockets-devel
+                            sudo dnf install -y pulseaudio-libs-devel
+                            sudo dnf install -y openssl-devel
+                            sudo dnf install -y libvorbis-devel
+                            sudo dnf install -y libwebp-devel
```

Comment:
then I can say to my mates at school I made a 100-200 line code script lol 
nah because I am a pain in my OWN arse 

though what I am more scared about is not having the packages on dnf ect ect and